src/rpdk/core/contract/resource_client.py

```diff
@@ -242,6 +278,33 @@ def assert_time(start_time, end_time, action):
             "Handler %r timed out." % action
         )
 
+    @staticmethod
+    def get_decoded_identifier_list(identifier_path):
+        list_identifier = [
+            identifier[len(identifier) - 1] for identifier in list(identifier_path)
```

Comment:
is the casting to list necessary here? is this really decoded or just the last element in the identifier ?

if this needs to stay, please use  `identifier[-1]` instead of `identifier[len(identifier) - 1]` to achieve consistency with the codebase. Also can just return the list immediately instead of assigning then returning

Reply:
updating it to return the value instead of storing it. Plus replaced ```identifier[len(identifier) - 1]``` with ```identifier[-1]```